doc/UsersGuide/source/SSP.rst

```diff
@@ -0,0 +1,176 @@
+.. index:: SSP
+
+SSP Support
+===========
+
+Composite models are imported and exported in the *System Structure Description (SSD)* format, which is part of the *System Structure and Parameterization (SSP)* standard.
+
+.. index:: SSP; Bus connections
+
+Bus Connections
+###############
+
+Bus connections are saved as annotations to the SSD file. Bus connectors are only allowed in weakly coupled and strongly coupled systems. Bus connections can exist in any system type. All atomic connections between atomic connectors must always exist. Bus connectors are simply used to hide atomic connections in the graphical interface. It is not required that all connectors referenced in a bus are connected. One bus may be connected to multiple other buses, and also to atomic connectors.
```

Comment:
I am not sure if the term `atomic` should be used here. I prefer `SSD connection` instead of `atomic connection`.

Change the sentence
`Bus connectors are simply used to hide atomic connections in the graphical interface.`
to
`Bus connectors are used to hide SSD connectors and bus connections are used to hide SSD connections in the graphical user interface.`

Reply:
Ok, I renamed the tag and changed from "atomic" to "SSD" in the text.

Reply:
I meant to replace `atomic` everywhere with `SSD`.

Reply:
Ok, done.